lib/api/src/rt/sys/entities/exception.rs

```diff
@@ -0,0 +1,85 @@
+//! Data types, functions and traits for `sys` runtime's `Tag` implementation.
+use std::any::Any;
+
+use wasmer_types::{TagType, Type};
+use wasmer_vm::StoreHandle;
+
+use crate::{
+    sys::vm::{VMException, VMExceptionRef},
+    AsStoreMut, AsStoreRef, Tag, Value,
+};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// A WebAssembly `exception` in the `sys` runtime.
+pub struct Exception {
```

Comment:
This should be pub(crate) I assume?

Reply:
[9a47e49](https://github.com/wasmerio/wasmer/pull/5347/commits/9a47e49caf04aad101ffcae627bce569623fe4ea)